docs/features/service_history.md

```diff
@@ -0,0 +1,190 @@
+# Service History
+
+The Service History subsystem provides a flattened, day-by-day representation of a client's journey through various programs. It is a primary source for some many reports and data analysis.
+
+## Key Models
+
+-   **`ServiceHistoryEnrollment`**: Each HUD enrollment produces one or more `ServiceHistoryEnrollment` rows that mark the lifecycle of the enrollment. We always create an `entry` record, we add an `exit` record when the enrollment closes, and we occasionally keep a `first` record for legacy compatibility. These markers allow downstream jobs to reason about the enrollment span independently of the daily service rows.
```

Comment:
The `first` record just marks the first time someone was seen in a homeless project.  I'm not sure where it is in use now, so maybe "legacy" is the right word here, but I don't think it has anything to do with compatibility.